### PR TITLE
set up front end extern and make library

### PR DESCRIPTION
**Pull request by danyal1234**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -83,5 +83,5 @@
         "locale": "cpp",
         "stack": "cpp"
     },
-    "python.pythonPath": "venv\\Scripts\\python.exe"
+    "python.pythonPath": "/usr/local/bin/python3"
```
Comment (joyglitch) [inline .vscode/settings.json:86]: Why is still uploaded? I swear this was in gitignore..

File: include/simulation.hh [hunk 2]
```diff
@@ -75,9 +75,41 @@ class Simulation {
...
+     * This get the current number of recovered agents
+     */
+    int getRecoveredTotal();
+
+    /**
+     * setInputs
+     * 
+     * Takes inputs from front end sends them to the associated classes
+     */
+    //int getRecoveredTotal(ask joyce for inputs after getting works);
```
Comment (joyglitch) [inline include/simulation.hh:112]: setInputs, not getRecoveredTotal :P

File: .vscode/settings.json (hunk 1 above)
Comment (danyal1234) [reply .vscode/settings.json:86]: no idea, even i git ignored it smh

File: include/simulation.hh (hunk 2 above)
Comment (danyal1234) [reply include/simulation.hh:112]: ya mb

**Pull request merged**